docs/articles/documentation/templates/authentication/role-options.md

```diff
@@ -0,0 +1,33 @@
+### options.preserveUrl
+
+Use this option to control which page is opened after you switch to the role.
+
+By default, TestCafe navigates back to the page that was opened previously to switching to the role.
```

Comment:
Change to:
By default, TestCafe navigates back to the previous page prior to the role change.

Reply:
I didn't mean that TestCafe navigates back prior to the role change. It's the page that was opened previously, navigation happens after the role activation.

*By default, TestCafe navigates back to the page opened before you switch to the role.*

Reply:
Can we say:
By default, TestCafe navigates back to the previous page prior to the role change.